utils/parseAttributes.ts

```diff
@@ -0,0 +1,15 @@
+export const parseAttributes = (
+  attributes: {
+    trait_type: string;
+    value: string;
+  }[],
+) => {
+  const parsedAttributes = attributes
+    .filter(({ value }) => value !== "")
```

Comment:
```suggestion
    .filter(({ value }) => value === 0 || value)
```

specifically check for `0` (which is valid) everything else (empty string, `null`, `undefined`) is invalid and should be filtered out

Reply:
with the single nft upload, 0 will actually be a string, so no need to check, example: https://thirdweb-r7lmo5xuh-thirdweb.vercel.app/dashboard/polygon/nft-drop/0x0dbC68329c91AAF7Bc849A27FF363DcBc38BBC55